### DISCUSSION TITLE
Dowload failed

### DISCUSSION THREAD

**Discussion by JuricaPOPCj** [aclap-dev/video-downloadhelper#1193] [Help and Q&A]
### Your question

After start the download stopped and ``download failed`` message is shown. Second problem: in Mozilla browser, Downloadhelper is not working on Zoom
  I have license paid

### Addon & CoApp version and details

Version 9.1.0.42 (stable), target mozilla, locale en-US.

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
           
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 142,
    "successfull_dl": 6,
    "never_show_successfull_dl_message": true,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": []
    }
}

**Comment by GrampaWildWilly** [comment]
You need to follow the instructions you ignored on the page where you created this problem report.  You need to include the entirety of the information from the `Copy information to clipboard` button in the VDH settings.  You also need to give at least one URL of a web page on which a video is hosted so others can try to replicate your problem.  To edit your post, look for the ... 3 dots above & to the right of your post.  Click the 3 dots.
